test/farm/helpers.bash

```diff
@@ -0,0 +1,22 @@
+# -*- bash -*-
+
+load ../system/helpers.bash
+
+function setup(){
+    # only set up the podman farm before the first test
+    if [[ "$BATS_TEST_NUMBER" -eq 1 ]]; then
```

Comment:
Instead of `BATS_TEST_NUMBER`, you can use `setup_suite()` and `teardown_suite()` functions in a file which must be named `setup_suite.bash`. See https://bats-core.readthedocs.io/en/stable/writing-tests.html#setup-and-teardown-pre-and-post-test-hooks

Reply:
Thanks, fixed